saleor/settings.py

```diff
@@ -556,3 +557,5 @@ def get_host():
 if REDIS_URL:
     CACHE_URL = os.environ.setdefault("CACHE_URL", REDIS_URL)
 CACHES = {"default": django_cache_url.config()}
+
+INVOICE_LOGO_PATH = os.environ.get("INVOICE_LOGO_PATH", "images/logo.svg")
```

Comment:
As @maarcingebala  said, we don't want to use saleor's logo by default.
And this will fail. How do you handle the scenario when a user doesn't want to add own logo? I see that `INVOICE_LOGO_PATH` requires path.